### ISSUE TITLE
Section 3.1.1.2: link key terms to glossary entries or explanations, and provide the latter.

### ISSUE THREAD

**Issue by jasonjgw** [w3c/maturity-model#161]
Key terms in section 3.1.1.2 should be explained or defined elsewhere, and links should connect the use of the terms to their explanations.

In particular, "accessibility statement", and possibly "statement of commitment to accessibility" should have associated explanations. In the former case, a link to any definition or explanation in European Union documentation would be useful, given the assertion about the legal requirement already in the text here.

**Comment by swinehartganderson** [comment]
This issue should be addressed in the most recent publication update. We hope you’ll review the updated information and continue providing feedback by filing Github issues.
